runtimes/lemonade/start.sh

```diff
@@ -0,0 +1,197 @@
+for model_config in models:
+    if model_config.get('provider') == 'lemonade':
+        print(model_config.get('lemonade', {}).get('port', ''))
+        break
+" 2>/dev/null)
+
+    CONFIG_BACKEND=$(uv run python -c "
+import yaml
+config = yaml.safe_load(open('$CONFIG_FILE'))
+models = config.get('runtime', {}).get('models', [])
+for model_config in models:
+    if model_config.get('provider') == 'lemonade':
+        print(model_config.get('lemonade', {}).get('backend', ''))
+        break
+" 2>/dev/null)
+
+    CONFIG_CONTEXT_SIZE=$(uv run python -c "
+import yaml
+config = yaml.safe_load(open('$CONFIG_FILE'))
+models = config.get('runtime', {}).get('models', [])
+for model_config in models:
+    if model_config.get('provider') == 'lemonade':
+        print(model_config.get('lemonade', {}).get('context_size', ''))
+        break
+" 2>/dev/null)
+
+    CONFIG_HF_TOKEN=$(uv run python -c "
+import yaml
+config = yaml.safe_load(open('$CONFIG_FILE'))
+models = config.get('runtime', {}).get('models', [])
+for model_config in models:
+    if model_config.get('provider') == 'lemonade':
+        print(model_config.get('huggingface_token', ''))
+        break
+" 2>/dev/null)
+
+
+    # Override defaults with config values if present
+    [ -n "$CONFIG_PORT" ] && LEMONADE_PORT="$CONFIG_PORT"
+    [ -n "$CONFIG_BACKEND" ] && LEMONADE_BACKEND="$CONFIG_BACKEND"
+    [ -n "$CONFIG_CONTEXT_SIZE" ] && LEMONADE_CONTEXT_SIZE="$CONFIG_CONTEXT_SIZE"
+    [ -n "$CONFIG_MODEL" ] && [ -z "$LEMONADE_MODEL" ] && LEMONADE_MODEL="$CONFIG_MODEL"
+
+    # Export Hugging Face token if provided (needed for downloading gated models)
+    [ -n "$CONFIG_HF_TOKEN" ] && export HF_TOKEN="$CONFIG_HF_TOKEN"
+fi
+
+# Color output for better UX
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+NC='\033[0m' # No Color
+
+echo -e "${GREEN}=== Lemonade Runtime Server ===${NC}"
+echo "Port: $LEMONADE_PORT"
+echo "Host: $LEMONADE_HOST"
+echo "Backend: $LEMONADE_BACKEND"
+
+# Check if lemonade-server-dev is available, install if not
+echo -e "${GREEN}Checking Lemonade SDK...${NC}"
+
+# Check if lemonade-sdk is installed
+if ! uv pip show lemonade-sdk &>/dev/null; then
+    echo -e "${YELLOW}Lemonade SDK not found. Installing...${NC}"
+    uv pip install lemonade-sdk || {
+        echo -e "${RED}Failed to install Lemonade SDK${NC}"
+        echo "Try running: uv pip install lemonade-sdk"
+        echo "For more info: https://lemonade-server.ai/"
+        exit 1
+    }
+    echo -e "${GREEN}Lemonade SDK installed successfully!${NC}"
+fi
+
+# Check if port is already in use
+if lsof -Pi :$LEMONADE_PORT -sTCP:LISTEN -t >/dev/null 2>&1 ; then
+    echo -e "${YELLOW}WARNING: Port $LEMONADE_PORT is already in use${NC}"
+    echo "Another Lemonade instance may be running, or the port is occupied."
+    echo ""
+    echo "Attempting to start anyway (will fail if port is truly unavailable)..."
+fi
+
+# Model is required for Lemonade
+if [ -z "$LEMONADE_MODEL" ]; then
```

Comment:
`LEMONADE_MODEL` is not optional per this code. That means, the earlier comments on this PR indiciating it is an error to omit `LEMONADE_MODEL` are correct.